user_docs/en/userGuide.t2t

```diff
@@ -2840,6 +2840,13 @@ If the add-on was previously "disabled", the status will show "enabled after res
 If the add-on was previously "enabled", the status will show "disabled after restart".
 Just like when you install or remove add-ons, you need to restart NVDA in order for changes to take effect.
 
++++ Reviewing and consulting feedback for add-ons +++[AddonStoreReviews]
+Before installing an add-on, you may want to see opinions and reactions by others.
+Also, you may want to provide feedback about add-ons to help other users.
+To see or share your feedback about add-ons, select an add-on and use the See or provide feedback option in the actions menu.
+This will open a dedicated webpage at GitHub Discussions, where you will be able to see or share your reactions for the add-on, or even for specific versions.
+Please, keep account that this doesn't replace direct communication with add-on developers. Instead, the purpose of this feature is to provide a mechanism to share feedback from the store to help users to decide if an add-on can be useful for them.

```

Comment:
"keep account" does not have meaning in English. I have adjusted to reach what I believe was your intended meaning.
Also, I have added a line break between the sentences, which translators like.

```suggestion
Please be aware that this doesn't replace direct communication with add-on developers.
Instead, the purpose of this feature is to provide a mechanism to share feedback from the store to help users decide if an add-on may be useful for them.
```